cogs/moderation.py

```diff
@@ -818,12 +818,19 @@ async def note(self, ctx, id: int, *, note):
         You must have the Moderator role to use this.
         """
 
+        
         result = await self.bot.mongo.db.action.find_one_and_update(
             {"_id": id, "guild_id": ctx.guild.id}, {"$set": {"note": note}}
         )
         if result is None:
             return await ctx.send("Could not find an entry with that ID.")
-        await ctx.send(f"Successfully added a note to that entry.")
+        if note.lower() == "reset":
+            await self.bot.mongo.db.action.find_one_and_update(
+            {"_id": id, "guild_id": ctx.guild.id}, {"$set": {"note": None}}
+        )
```

Comment:
```suggestion
        if note.lower() == "reset":
            await self.bot.mongo.db.action.find_one_and_update(
                {"_id": id, "guild_id": ctx.guild.id}, {"$unset": {"note": 1}}
            )
```

Fix indentation, also use unset.

Reply:
Oh okay, thankies.